docs/pages/includes/role-spec.mdx

```diff
@@ -141,6 +141,9 @@ spec:
       # 'region': '^us-west-1|eu-central-1$'
       'reg': '^us-west-1|eu-central-1$'
 
+    # List of host groups the created user will be added to. Any that don't already exist are created.
```

Comment:
Should we say something like "only applies when create_host_user_mode is not off" ??

Reply:
Yeah that makes sense to me :+1: I updated the `rbac` reference to show the same